lib/environments/meteorEnvironment.js

```diff
@@ -321,13 +321,7 @@ export default {
     ['.html', '.css'].forEach((ext: string) => {
       const moduleSpecifier = `${basePath}${ext}`;
       if (fs.existsSync(path.join(config.workingDirectory, moduleSpecifier))) {
-        if (config.get('useRelativePaths')) {
-          moduleSpecifiers.push('.'.concat(path.sep, path.basename(moduleSpecifier)));
-        } else {
-          // Strip the leading '.' off of the moduleSpecifier to turn it into a Meteor compliant
-          // absolute path.
-          moduleSpecifiers.push(moduleSpecifier.slice(1));
```

Comment:
I'm a little iffy about removing useRelativePaths. I agree that it can be implemented within moduleNameFormatter. My problem with doing that is that it seems like something that is too common for a user to want to change. If they change it, they'll be burdened not just with implementing that in their user config, but also with carrying over anything else the environment implemented in moduleNameFormatter.

I'm probably coming from an unusual perspective though because npm packages likely almost always use relative. Meteor has started off relative, but there are some big groups leaning hard towards absolute.

When writing an app using ES6 modules, the need for local packages is gone, modularization can be accomplished within the app without packages. This then leads you to larger directory structures. Users are finding it more convenient to use absolute paths because when they decide to juggle that directory structure up, they can easily do a global search and replace on an absolute reference. That can't be done with relative references.